### ISSUE TITLE
Reconnection handles and bend points are off

### ISSUE THREAD

**Issue by ugurdogrusoz** [iVis-at-Bilkent/pathway-mapper#207]
Reconnection handles and bend points are off. In lower zoom levels, they are drawn off their actual coordinates:
![image](https://user-images.githubusercontent.com/3874988/66698827-fdbbf700-ece9-11e9-9a3d-40c0265052a3.png)

**Comment by ugurdogrusoz** [comment]
Similar problem with resize handles:
<img width="623" alt="Screenshot 2019-10-23 at 08 28 07" src="https://user-images.githubusercontent.com/3874988/67360654-2d32f500-f56f-11e9-98cb-9167bc437209.png">

**Comment by msalihaltun** [comment]
I want to comment on this for future reference, and also since I'm actually kind of amazed.

The node-editing canvas was off center relative to the cytoscape canvas because the border that we use for cytoscape was defined as a style property of the cytoscape `div` itself. Cytoscape respected the border and started drawing from inside it, but Konva was given the cytoscape root `div` as source in `cytoscape-node-editing` (naturally). This caused every element drawn on the Konva canvas to be off center by exactly the size of the border we defined for cytoscape. Basically, if Konva canvas started at `(0,0)`, the cytoscape canvas would start at `(border, border)`. Everything we positioned at Konva canvas would be off centered. 

The solution is to use a wrapper div for the cytoscape canvas, and place the border there (or just remove the border, but that doesn't look so good).
